src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="crossorigin"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <a href="#home" class="navImage"><img src="./images/logo.png" alt="navImage"></a>
+
+      <nav class="navMenu">
+        <ul class="navList">
+          <li><a href="#apple" class="navLink is-active">Apple</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#apple" class="navLink is-active">
            Apple
          </a>
        </li>
```

исправь везде стиль кода